src/profiler/profiler.h

```diff
@@ -1193,7 +1212,11 @@ struct ProfileOperator : public ProfileEvent {
       if (attributes) {
         name_.append(attributes->to_string().c_str());
       }
-      categories_.set("operator");
+      if (IsSubOperatorOfCustom(const char* name)) {
```

Comment:
You can't write a function definition in if clause, right? :)

Reply:
Copy-pasted the code... I apologize. Will pay more attention